CHANGES.md

```diff
@@ -4365,10 +4358,10 @@ OpenSSL 1.0.2
 
    *Matt Caswell*
 
- * Fix memory issues in BIO_*printf functions
+ * Fix memory issues in `BIO_*printf` functions
 
    The internal |fmtstr| function used in processing a "%s" format string in
-   the BIO_*printf functions could overflow while calculating the length of a
+   the `BIO_*printf` functions could overflow while calculating the length of a
    string and cause an OOB read when printing very long strings.
 
    Additionally the internal |doapr_outch| function can attempt to write to an
```

Comment:
There are a lot of places where pipes are used as delimiters for code code, like ` |doapr_outch|`.  Mind replacing them with backticks? For example, this change note contains a lot of those:

```
 * Fix BN_hex2bn/BN_dec2bn NULL pointer deref/heap corruption

   In the BN_hex2bn function the number of hex digits is calculated using an
   int value |i|. Later |bn_expand| is called with a value of |i * 4|. For
   large values of |i| this can result in |bn_expand| not allocating any
   memory because |i * 4| is negative. This can leave the internal BIGNUM data
   field as NULL leading to a subsequent NULL ptr deref. For very large values
   of |i|, the calculation |i * 4| could be a positive value smaller than |i|.
   In this case memory is allocated to the internal BIGNUM data field, but it
   is insufficiently sized leading to heap corruption. A similar issue exists
   in BN_dec2bn. This could have security consequences if BN_hex2bn/BN_dec2bn
   is ever called by user applications with very large untrusted hex/dec data.
   This is anticipated to be a rare occurrence.
```